### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by neotheprogramist**

### CODE REVIEW

File: src/deserialization/stark.cairo
Comment (Okm165) [inline src/deserialization/stark.cairo:1]: dont u want to split this file into smaller segments like stark public-input pow?

**Review by Okm165 — APPROVED** [review]
lgtm

File: src/deserialization/stark.cairo
Comment (neotheprogramist) [reply src/deserialization/stark.cairo:1]: fixed pow, I've left public input in the same file as it is the direct member of `StarkProofWithSerde` struct

File: src/air/config.cairo
Comment (tiagofneto) [inline src/air/config.cairo:1]: Can we rename this file to `traces_config` or something along those lines, for it not to be confused with the actual AIR config

**Review by tiagofneto — APPROVED** [review]

File: src/air/config.cairo
Comment (neotheprogramist) [reply src/air/config.cairo:1]: fixed

**Pull request merged**